packages/dds/matrix/package.json

```diff
@@ -11,74 +11,66 @@
 	"license": "MIT",
 	"author": "Microsoft and contributors",
 	"sideEffects": false,
+	"type": "module",
 	"exports": {
 		".": {
 			"import": {
-				"types": "./lib/index.d.mts",
-				"default": "./lib/index.mjs"
+				"types": "./lib/index.d.ts",
+				"default": "./lib/index.js"
 			},
 			"require": {
 				"types": "./dist/index.d.ts",
 				"default": "./dist/index.js"
 			}
 		},
-		"./alpha": {
+		"./public": {
 			"import": {
-				"types": "./lib/matrix-alpha.d.mts",
-				"default": "./lib/index.mjs"
+				"types": "./lib/matrix-public.d.ts",
+				"default": "./lib/index.js"
 			},
 			"require": {
-				"types": "./dist/matrix-alpha.d.ts",
+				"types": "./dist/matrix-public.d.ts",
 				"default": "./dist/index.js"
 			}
 		},
-		"./beta": {
+		"./alpha": {
 			"import": {
-				"types": "./lib/matrix-beta.d.mts",
-				"default": "./lib/index.mjs"
+				"types": "./lib/matrix-alpha.d.ts",
+				"default": "./lib/index.js"
 			},
 			"require": {
-				"types": "./dist/matrix-beta.d.ts",
+				"types": "./dist/matrix-alpha.d.ts",
 				"default": "./dist/index.js"
 			}
 		},
 		"./internal": {
 			"import": {
-				"types": "./lib/index.d.mts",
-				"default": "./lib/index.mjs"
+				"types": "./lib/index.d.ts",
+				"default": "./lib/index.js"
 			},
 			"require": {
 				"types": "./dist/index.d.ts",
 				"default": "./dist/index.js"
 			}
-		},
-		"./public": {
-			"import": {
-				"types": "./lib/matrix-public.d.mts",
-				"default": "./lib/index.mjs"
-			},
-			"require": {
-				"types": "./dist/matrix-public.d.ts",
-				"default": "./dist/index.js"
-			}
 		}
 	},
 	"main": "dist/index.js",
-	"module": "lib/index.mjs",
 	"types": "dist/index.d.ts",
 	"scripts": {
 		"api": "fluid-build . --task api",
-		"api-extractor:commonjs": "api-extractor run --local",
-		"api-extractor:esnext": "api-extractor run --config ./api-extractor-esm.json",
+		"api-extractor:commonjs": "api-extractor run --config ./api-extractor-cjs.json",
+		"api-extractor:esnext": "api-extractor run --local",
 		"bench": "cd bench && node --expose-gc -r ts-node/register src/index.ts",
-		"bench:profile": "cd bench && tsc && node -r ts-node/register --prof --logfile=profile.log --no-logfile-per-isolate src/index.ts --runInBand && node --prof-process profile.log > profile.txt && rimraf profile.log && echo See results in bench/profile.txt",
+		"bench:profile": "cd bench && node -r ts-node/register --prof --logfile=profile.log --no-logfile-per-isolate src/index.ts --runInBand && node --prof-process profile.log > profile.txt && rimraf profile.log && echo See results in bench/profile.txt",
```

Comment:
Having 'tsc' in the 'bench:profile' script caused policy-check to panic due to missing task dependencies.